packages/xo-web/src/common/intl/messages.js

```diff
@@ -1460,6 +1460,8 @@ const messages = {
     'Are you sure you want to delete all the backups from {nMetadataBackups, number} metadata backup{nMetadataBackups, plural, one {} other {s}}?',
   bulkDeleteMetadataBackupsConfirmText:
     'delete {nMetadataBackups} metadata backup{nMetadataBackups, plural, one {} other {s}}',
+  remoteNotLinkedToSelectedProxy:
+    'Remote not linked to the selected proxy, it will be skipping',
```

Comment:
*skipped*?

Reply:
And this has not been decided yet, it may simply not work.

Reply:
True :+1:  should i remove the second sentence? Because i don't know the future behavior. 
I'll update the message if needed when implementing the solution.

Reply:
This remote is not associated with the selected proxy.
or 
This remote is not linked to the selected proxy.
?